book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
```

Comment:
- [ ] We should rearrange this arguments to match how the Rental constructor is defined, like so:

  ```
  Rental.new(date, self, person)
  ```